src/main/java/org/kiwiproject/eureka/EmbeddedEurekaServer.java

```diff
@@ -0,0 +1,101 @@
+package org.kiwiproject.eureka;
+
+import static com.google.common.base.Preconditions.checkState;
+
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.jetty.server.Connector;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.webapp.WebAppContext;
+import org.kiwiproject.io.KiwiPaths;
+
+/**
+ * Creates, configures, starts and stops an embedded Jetty server running a Eureka Server.
+ */
+@Slf4j
+public class EmbeddedEurekaServer {
+
+    private static final long IDLE_TIMEOUT = 3_600_000L;
```

Comment:
How about `TimeUnit.HOURS.toMillis(1L)` so I don't have to try and do this math in my head?

Reply:
Also suggest renaming to `IDLE_TIMEOUT_MILLIS` for the ultimate in clarity 🤘